src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -2168,6 +2168,24 @@ void M(String parameter)
             await VerifyItemIsAbsentAsync(markup, "parameter");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(53585, "https://github.com/dotnet/roslyn/issues/53585")]
+        public async Task AfterStaticLocalFunction_TypeOnly()
+        {
+            var markup = @"
+using System;
+class C
+{
+    void M(String parameter)
+    {
+        static $$
+    }
+}
+";
+            await VerifyItemExistsAsync(markup, "String");
```

Comment:
Is `string` offered too? It has a separate keyword recommender IIRC (there are other keywords too).

Reply:
It wasn't offered. I fixed this and added tests.